packages/controls-navigator/src/NavigatorControl.vue

```diff
@@ -0,0 +1,293 @@
+<template>
+  <div :class="getNavigateClass()">
+    <v-stepper :model-value="currentStep - 1">
+      <template v-for="index in steps" :key="index">
+        <v-stepper-item
+          :editable="navigationType === 'tab' || currentStep > index"
+          :title="prompts[index - 1] ? prompts[index - 1].name : ''"
+          :complete="getComplete(index)"
+          :class="getStepClass(currentStep, index)"
+          @click="gotoStep(index)"
+        />
+        <div class="breadcrumbs-container">
+          <YOUIBreadcrumbs
+            :id="`breadcrumbs-${index}`"
+            class="breadcrumbs"
+            :breadcrumbs="answers[prompts[index - 1] ? prompts[index - 1].name : undefined]"
+          />
+        </div>
+      </template>
+    </v-stepper>
+  </div>
+</template>
+
+<script setup>
+import { computed, ref, toRefs, unref, watchEffect } from "vue";
+import YOUIBreadcrumbs from "./YOUIBreadcrumbs.vue";
+import * as _ from "lodash";
+const props = defineProps({
+  promptIndex: {
+    type: Number,
+    default: 0,
+  },
+  prompts: {
+    type: Array,
+    default: () => [],
+  },
+  promptAnswers: {
+    type: Object,
+    default: () => {},
+  },
+  allAnswers: {
+    type: Object,
+    default: () => {},
+  },
+  navigationType: {
+    type: String,
+    default: "tab",
+  },
+});
+
+const { promptIndex, prompts, allAnswers, promptAnswers, navigationType } = toRefs(props);
+
+const emits = defineEmits(["onGotoStep"]);
+
+const answers = ref({});
+
+const currentStep = ref(0);
+
+const travel = ref({});
```

Comment:
can u add a comment what is the purpose of the `travel`?

Reply:
travel is used to keep track of the steps that have been visited